magstock/templates/regextra.html

```diff
@@ -2,7 +2,9 @@
 <script type="text/javascript">
     if (window.BADGE_TYPES) {
         BADGE_TYPES[0].description = 'Includes three day admission and campground fees';
```

Comment:
This is an unrelated fix for a JS error I noticed while I was testing this.  We should only update the description of the supporter badge type if that badge type actually exists.